fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/DownloadTaskRequest.kt

```diff
@@ -18,8 +18,9 @@ internal class DownloadTaskRequest(request: Request) : TaskRequest(request) {
         response.dataStream.copyTo(fileOutputStream, BUFFER_SIZE) { readBytes ->
             progressCallback?.invoke(readBytes, response.contentLength)
         }
+        fileOutputStream.close()
         return response
     }
 }
 
-private const val BUFFER_SIZE = 1024
\ No newline at end of file
+private const val BUFFER_SIZE = 1024
```

Comment:
Maybe take the chance and add a newline at the end of the file?

Reply:
Actually, that was apparently already done by GitHub's online editor (I was doing this change online).

Reply:
Nice.